app/models/loader/orchestrate.rb

```diff
@@ -120,8 +120,9 @@ def delete_shadowed_and_duplicate_rows
     end
 
     # TODO: consider renaming this method
-    def store_policy_in_db
-      eliminate_duplicates_pk
+    def store_policy_in_db(reject_duplicates: false)
+      removed_duplicates_count = eliminate_duplicates_pk
+      raise ApplicationController::BadRequest, "Updating existing resource disallowed in additive policy operation" if removed_duplicates_count.positive? && reject_duplicates
```

Comment:
It's better to raise a local exception to avoid coupling the loader to the somewhat global `ApplicationController`. Have a look at `Exceptions::InvalidPolicyObject` for inspiration, and see how it is rescued from in `ApplicationController`, see 

https://github.com/cyberark/conjur/blob/956e8b65878ddd5722ce9c060ff5b4609157990c/app/controllers/application_controller.rb#L61

Reply:
Created `Exceptions::DisallowedPolicyOperation`